packages/block-editor/src/components/block-title/index.js

```diff
@@ -73,5 +76,14 @@ export default function BlockTitle( { clientId } ) {
 	if ( label !== blockType.title ) {
 		return truncate( label, { length: 35 } );
 	}
-	return blockInformation.title;
+	return createInterpolateElement(
+		sprintf(
+			/* translators: 1: The block title. */
+			__( '%s <VisuallyHidden>Block.</VisuallyHidden>' ),
```

Comment:
Hiding the word "Block" visually might not work with speech-to-text, because the word is still part of the button label. Assuming speech-to-text software reads labels either from the DOM tree, or from the accessibility API, it won't be able to tell that the word is hidden and may expect users to read out the whole label in order to activate the button.

Would we lose much by dropping the word "block" altogether? Users entering navigation mode should already be focused in the content area, so would expect to be interacting with blocks. Or am I missing something?

Reply:
I added the extra word "block" for 2 reasons.
1. It exists in the current aria-label.
2. I believe it adds context to what the element is. The problem in Gutenberg I feel sometimes is lack of context, this was meant to take away any confusion.

I do understand your point about text to speech software though, this should be tested.

Thanks.

Reply:
Just a headsup that the block title (which moved here) isn't read on [mobile](https://github.com/WordPress/gutenberg/tree/master/packages/react-native-editor). It looks like <BlockTitle> isn't used they same way on mobile as its text is not read by VoiceOver (I gave this a test on iOS). I haven't got full clarity on what's going on here, but wanted to share.
Aside, it might be best to conditionally on mobile return `blockInformation.title` (as was done previously here) since `createInterpolateElement` seems web-specific.

Reply:
I eliminated title changes from this PR.